docs/examples/Generic Branch Example.ipynb

```diff
@@ -796,7 +779,7 @@
     "This chapter provides a guide on how to use the Generic Branch to import data from CIM/CGMES network datasets and integrate it into a PGM model.\n",
```

Comment:
maybe out of scope for this PR but it feels to me like this section should go to PGM-IO instead. See e.g. our example on importing data from Arrow in https://power-grid-model-io.readthedocs.io/en/stable/examples/arrow_example.html , which we technically do not have an off-the-shelf conversion function for, but we do have some documentation on how to handle it.

That said, this is really useful documentation! Thank you for contributing this!

Reply:
definitely not! only this section on CIM/CGMES conversion probably belongs in PGM-IO. The rest of the document that shows how the Generic Branch component works is definitely in the right place as it is.

Reply:
we can also decide to move it later

Reply:
it is of high interest 😉 we are extremely happy with this and @petersalemink95 already forwarded it internally. it's mainly the purpose of PGM vs the purpose of PGM-IO that makes this a little bit tricky. We'll discuss it offline and do a follow-up PR if necessary. Again, thanks for the input!